src/common/scss/form-input.scss

```diff
@@ -79,6 +80,15 @@ textarea {
   color: var(--kd-color-text-error);
   margin-top: 8px;
 
+  &.error-text {
+    font-size: 16px;
```

Comment:
Shouldn't need to specify this font size, it's the default body font size.

Reply:
✅